typos.toml

```diff
@@ -0,0 +1,12 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md to configure typos
```

Comment:
Move this file to the .github directory.

Reply:
Why? It is supposed to be placed in the ROOT.

Reply:
When you run typos locally, it will search `typos.toml` in the current directory and use it.

Reply:
Because I try to keep the root directory thin as readers don't get values from seeing it. The `.all-contributorsrc` and `.golangci-lint` will be moved as well when I get to this.

Reply:
Or rename it as `.typos.toml`?

Reply:
> Because I try to keep the root directory thin as readers don't get values from seeing it. The `.all-contributorsrc` and `.golangci-lint` will be moved as well when I get to this.

I see. Do you mind it if I add a Makefile target?

Reply:
But we don't have or use Makefile in this project. Are there any problems with storing this file inside the `.github` directory?

Reply:
It 's not about github actions, but about locally run binaries. I need to know where the config file is and specify them. So it's better to add it to Makefile.

Reply:
But I get what you mean. Let me do it.

Reply:
> It's not about github actions, but about locally run binaries.

I appreciate the help, but I'm not going to run this locally. 

Thinking of it now - don't we have something like this in golangci?

Reply:
> > It's not about github actions, but about locally run binaries.
> 
> I appreciate the help, but I'm not going to run this locally.
> 
> Thinking of it now - don't we have something like this in golangci?

I see.